components/src/core/components/Input/password-input.scss

```diff
@@ -0,0 +1,123 @@
+.password-container {
+  position: relative;
+  width: 100%;
+  .password-strength-check {
+    position: absolute;
+    right: 0;
+    top: 0;
+    border-radius: 0.65rem;
+    margin-top: -25px;
+    font-family: Nunito;
+    font-weight: 700;
+    font-size: 12px;
+    color: white;
+    padding: 2px 10px;
+  }
+
+  .password-view-icon-container {
+    position: absolute;
+    border-radius: 0.65rem;
+    right: -0.18rem;
+    height: 2.2rem;
+    width: 2.2rem;
+    top: 0.32rem;
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-pack: center;
```

Comment:
Do not add [vendor prefixes](https://developer.mozilla.org/en-US/docs/Glossary/Vendor_Prefix) they will be auto added by postcss plugin

Reply:
remove these